ui/org.eclipse.pde.launching/src/org/eclipse/pde/internal/launching/JUnitLaunchRequirements.java

```diff
@@ -0,0 +1,164 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
```

Comment:
The first year should be the one, where the contained code was added first. But from the git-history, at least some methods where there in the first git commit in 2009, so I guess the year 2006 of the source file is the original date:
```suggestion
 * Copyright (c) 2006, 2025 IBM Corporation and others.
```